shopify/resources/application_credit.py

```diff
@@ -0,0 +1,12 @@
+from ..base import ShopifyResource
+
+class ApplicationCredit(ShopifyResource):
+    _prefix_source = "application_credits/$application_credit_id/"
+
+    @classmethod
+    def _prefix(cls, options={}):
+        application_credit_id = options.get("application_credit_id")
+        if application_credit_id:
+            return "%s/application_credits/%s" % (cls.site, application_credit_id)
+        else:
+            return "%s" % (cls.site)
```

Comment:
tbh I'm not sure if this part is right... this was how I got it to return the site with `/application_credits` tacked onto the end

Reply:
Overloading the `_prefix` isn't necessary here, it's only needed for resources that can be conditionally accessed at different routes, e.g. variants can be accessed from `/admin/#{product_id}/#{variant_id}`, or simply `/admin/#{variant_id}`.